openedx/core/djangoapps/content/course_structures/signals.py

```diff
@@ -1,29 +0,0 @@
-"""
-Django Signals classes and functions for the Course Structure application
-"""
-from django.dispatch.dispatcher import receiver
-
-from xmodule.modulestore.django import SignalHandler
-
-from .models import CourseStructure
-
-
-@receiver(SignalHandler.course_published)
-def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=unused-argument
-    """
-    Course Structure application receiver for the course_published signal
-    """
-    # Import tasks here to avoid a circular import.
-    from .tasks import update_course_structure
-
-    # Delete the existing discussion id map cache to avoid inconsistencies
```

Comment:
Was this handled appropriately in @ormsbee's earlier work to prep for the discussion move?

Reply:
Yes.  The new [update_mapping](https://github.com/edx/edx-platform/blob/6a4924f9d42baa5eb1dfa39f2d02ea4f7f683851/common/djangoapps/django_comment_common/models.py#L217) code calls `get_or_create` and updates appropriately.